test/packages/kubernetes/data_stream/pod/fields/ecs.yml

```diff
@@ -7,3 +7,7 @@
 - name: service.type
   type: keyword
   description: Service type
+- name: orchestrator.cluster.name
```

Comment:
To enable dependency management here, you need to include [build.yaml](https://github.com/elastic/integrations/blob/master/packages/mysql/_dev/build/build.yml) file.

Reply:
So I will just add this file

Reply:
You need to format it with `elastic-package format`:

> [2021-07-30T13:08:28.902Z] Error: checking package failed: formatting the integration failed (path: /var/lib/jenkins/workspace/t-manager_elastic-package_PR-452/src/github.com/elastic/elastic-package/test/packages/kubernetes, failFast: true): walking through the integration files failed: formatting file failed (path: /var/lib/jenkins/workspace/t-manager_elastic-package_PR-452/src/github.com/elastic/elastic-package/test/packages/kubernetes/_dev/build/build.yml): file is not formatted (path: /var/lib/jenkins/workspace/t-manager_elastic-package_PR-452/src/github.com/elastic/elastic-package/test/packages/kubernetes/_dev/build/build.yml)